packages/select/docs/MenuLoadingSkeleton.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import { Meta, StoryObj } from "@storybook/react"
+import { ComponentDocsTemplate } from "../../../storybook/components/DocsContainer"
+import { MenuLoadingSkeleton } from "../src/FilterMultiSelect/components/MenuLayout/MenuLoadingSkeleton/index"
+
+const meta = {
+  tags: ["autodocs"],
+  title: "Components/Filter Multi-Select/Menu Loading Skeleton",
+  component: MenuLoadingSkeleton,
+  args: {
+    isAnimated: false,
+  },
+  parameters: {
```

Comment:
Ah apologies, you'll need to add a `installation` key here, for example:
```
    installation: [
      "npm install @kaizen/button",
      "import { Button } from `@kaizen/button`",
    ],
```

Reply:
I had this one but removed it because it looked a little confusing, i.e. it ends up being 
```
    installation: [
      "npm install @kaizen/select",
      "import { FilterMultiSelect } from `@kaizen/select`",
    ],
```

is it better to have it regardless?

Reply:
i've pushed it for now, guess it still can't hurt to know where it exists

Reply:
Yeah, we're noting these things to optimise in the next iteration of docs 😄